exporters/metrics/util/src/main/java/io/opencensus/exporter/metrics/util/IntervalMetricReader.java

```diff
@@ -39,7 +39,9 @@
 
   private IntervalMetricReader(Worker worker) {
     this.worker = worker;
-    this.workerThread = new DaemonThreadFactory().newThread(worker);
```

Comment:
Do we need this change? It looks to me nothing changed for JDK 8+. The previous approach only restricts JDK 7.

Reply:
Previous approach created thread using MoreExecutors.platformThreadFactory() for all JDK. We just need regular thread for logging to work.